simplq/src/styles/homePage.module.scss

```diff
@@ -1,41 +1,54 @@
 @import 'utils';
 
-div {
-  @include div-padding();
-}
-
```

Comment:
We should not do this globally, would give unwanted side effects, so in our case, the text box from material ui gets unwanted padding becasue of this:

![image](https://user-images.githubusercontent.com/11256376/93669590-e61cef00-fab2-11ea-9b9c-37ba41a2e852.png)